app.config.js

```diff
@@ -4,6 +4,7 @@ module.exports = {
   // networks in their wallet.
   // Ocean Protocol contracts are deployed for: 'mainnet', 'rinkeby', 'development'
   network: process.env.GATSBY_NETWORK || 'mainnet',
+  rbacUrl: process.env.RBAC_URL || 'false',
```

Comment:
1. if you want to have access to this env var in the browser it needs to be prefixed with `GATSBY_`, [see here](https://www.gatsbyjs.com/docs/how-to/local-development/environment-variables/#client-side-javascript)
2. that `'false'` becomes redundant because if `process.env.RBAC_URL` is undefined it's enough to know that it's not there

Reply:
1. That's interesting because it seemed to be working ok in the browser. I have changed it to `GATSBY_RBAC_URL` now though:
https://github.com/oceanprotocol/market/blob/28e1b912c5c0c786d440b5f3b8e543b1a08431f8/app.config.js#L7

2. I've removed the `false` and am checking for undefined now: 
https://github.com/oceanprotocol/market/blob/28e1b912c5c0c786d440b5f3b8e543b1a08431f8/src/utils/rbac.ts#L9